include.js

```diff
@@ -407,6 +407,13 @@ let cards = [
     author: 'Sneha Das',
     githubLink: 'https://github.com/iamsnehadas'
   },
+  {
+    artName: '3D toaster Animation', 
+    pageLink: './Art/https://github.com/apu52/index.html', 
+    imageLink: './Art/https://github.com/apu52/3D Toaster.gif', 
+    author: 'Arpan Chowdhury', 
+    githubLink: 'https://github.com/https://github.com/apu52' 
```

Comment:
# Feedbacks on include js content
- [ ] invalid githubLink url > `https://github.com/https://github.com/apu52`
- [ ] invalid imageLink > this does not convey with a folder name you added a link which in incorrect
- [ ] invalid pageLink > this does not convey with a folder name you added a link which in incorrect
Please check the example on the readme.md file and correctly modify your PR 
--- 
also note that your file must not have any space character in their names
--> this should includes modifications based on your folder structure and file renaming